wfl/Database/MongoDBManager.cs

```diff
@@ -8,55 +8,31 @@
 
 namespace Symphono.Wfl.Database
 {
-    public class MongoDBManager: IDBManager
+    public class MongoDBManager<T>: IDBManager<T> where T: IEntity
     {
         private static string connectionString;
         public static MongoClient client;
         public static IMongoDatabase db;
+        private string collectionName { get; }
```

Comment:
I was under the impression that using `{ get; }` made the variable readonly - is this not the case?